reference/api/service_invocation_api.md

```diff
@@ -72,12 +72,12 @@ curl http://localhost:3500/v1.0/invoke/mathService/method/add \
   -d '{ "arg1": 10, "arg2": 23}'
 ```
 
-The `mathService` service will need to be listening on the `/add` endpoint to receive and process the request.
+The `mathService` service will need to be listening on the `/v1.0/add` endpoint to receive and process the request.
```

Comment:
this doesnt need a v1.0 as its the service method.

Reply:
I thought we agreed on this - https://github.com/dapr/dapr/issues/1678#issuecomment-642986030 

It would be good to have a meeting to discuss it.